trunk/SUAVE/Methods/Missions/Segments/Descent/Constant_Speed_Constant_Angle_Noise.py

```diff
@@ -57,7 +57,8 @@ def expand_state(segment,state):
     
     state.numerics.number_control_points = n_points
     
-    state.expand_rows(n_points)      
+    # MADE n_points an integer.......dont know why it wasnt.....please help me ............... 
```

Comment:
It was supposed to be an int before... I remember that this was fixed by me at some point. Maybe in a different part of the code.

Reply:
I couldn't figure out why it stopped working.

Reply:
Ok, here's why:
It was fixed for climb:
https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Methods/Missions/Segments/Climb/Constant_Speed_Constant_Angle_Noise.py#L50

But not for descent. And only just now did you add in the approach noise in the regression